src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,11 @@
+databaseChangeLog:
```

Есть ещё одно предположение... Можно для себя проверить, что в конце пути указан символ `/`, указывающий, что смотреть надо на директорию, а не конкретный файл, либо указать путь не относительно `db.changelog-master.yaml`, а от папки `resources` (`db/changelog/0.1/`), заодно убрав соответствующий параметр (`relativeToChangeLogFile`)
```suggestion
databaseChangeLog:
  - includeAll:
      path: 0.1/
      relativeToChangeLogFile: true
      errorIfMissingOrEmpty: true
```